bbchallenge-paper.tex

```diff
@@ -309,19 +309,19 @@ \subsection{Main Result}\label{sec:intro:mainresults}
 
 
 
-\paragraph{Challenges.} Proving $S(5) = \BBtheFifth$ required analyzing the behavior of $\BBtheFifthTNF$ Turing machines\footnote{This exact number can only be known after the proof is done: it is as hard as computing $S(5)$, see Section~\ref{sec:enum}.}-- evidently requiring computer assistance. The challenge for analysing a halting machine is the case where it halts after a number of steps that is too enormous to be simulated step-by-step (e.g. the current 6-state champion halts after $10 \uparrow \uparrow 15$ steps \cite{Pavel_discorvery}, see Appendix~\ref{app:lowerbounds}), this challenge was not met for 5-state machines since they halt in at most $\BBtheFifth$ steps -- this could not have been known for certain in advance but was believed -- which is easy to simulate on modern computers. The challenge for analysing a nonhalting machine is that proving that it does not halt can be hard. How hard?
+\paragraph{Challenges.} Proving $S(5) = \BBtheFifth$ required analyzing the behavior of $\BBtheFifthTNF$ Turing machines\footnote{This exact number can only be known after the proof is done: it is as hard as computing $S(5)$, see Section~\ref{sec:enum}.}-- evidently requiring computer assistance. The challenge for analysing a halting machine is the case where it halts after a number of steps that is too enormous to be simulated step-by-step (e.g. the current 6-state champion halts after $10 \uparrow \uparrow 15$ steps \cite{Pavel_discorvery}, see Appendix~\ref{app:lowerbounds}); this challenge did not end up being relevant for 5-state machines, since they halt in at most $\BBtheFifth$ steps, which is easy to simulate on modern computers. The challenge for analysing a nonhalting machine is that proving that it does not halt can be hard. How hard?
```

Comment:
Change not approved, (a) wording modified in previous PR, (b) the deleted sentence is important